devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -3,25 +3,28 @@
 // driver for SmartThings
 // The EZMultiPli is also known as the HSM200 from HomeSeer.com
 
+import physicalgraph.zwave.commands.*
+
 metadata {
-	definition (name: "EZmultiPli", namespace: "DrZWave", author: "Eric Ryherd", oauth: true) {
```

Comment:
what does this `oauth` flag do?

Reply:
Nothing, I think. I think it might have existed in error. There is an oauth flag for SmartApps.